src/coffee/directives/api/models/parent/circle-parent-model.coffee

```diff
@@ -3,6 +3,12 @@ angular.module('uiGmapgoogle-maps.directives.api.models.parent')
 ['uiGmapLogger', '$timeout','uiGmapGmapUtil',
 'uiGmapEventsHelper', 'uiGmapCircleOptionsBuilder',
 ($log, $timeout, GmapUtil, EventsHelper, Builder) ->
+  _settingFromDirective = (scope, fn) ->
```

Comment:
This is the main important boolean gate addition to not have recursive updates from google -> scope -> google -> scope ..etc